velox/common/memory/MmapAllocator.cpp

```diff
@@ -26,8 +26,14 @@ MmapAllocator::MmapAllocator(const Options& options)
     : kind_(MemoryAllocator::Kind::kMmap),
       useMmapArena_(options.useMmapArena),
       capacity_(bits::roundUp(
-          options.capacity / AllocationTraits::kPageSize,
-          64 * sizeClassSizes_.back())) {
+          options.capacity * (100 - options.smallAllocationReservePct) / 100 /
```

Comment:
use: AllocationTraits::numPages

Reply:
I think it might not be applicable here because it is trying to round up by factor of (64 * sizeClassSizes_.back())